### PR TITLE
docs: relayer setup and usage

### PR DESCRIPTION
**Pull request by conorsch**
These docs changes are based on feedback from a pairing session with @aubrika today.

### CODE REVIEW

File: deployments/relayer/build-path [hunk 1]
```diff
@@ -27,6 +27,6 @@ visible_log "These commands should all yield results:"
 
 cat <<EOF
   * pcli -n http://localhost:8080 q ibc client 07-tendermint-0
-  * pcli -n http://localhost:8080 q ibc channel transfer channel-0
+  * pcli -n http://localhost:8080 q ibc channel --channel-id 0
```
Comment (hdevalence) [inline deployments/relayer/build-path:30]: why does this command have different syntax to all of the other `q ibc` commands?

File: docs/guide/src/pcli/transaction.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ while read -r i ; do
         echo "Most recently active channel is: $(( i - 1))" ;
         break ;
     fi ;
-done <<< "$(seq 0 10)"
+done <<< "$(seq 0 100)"
```
Comment (hdevalence) [inline docs/guide/src/pcli/transaction.md:201]: side note, but why don't we just do the real query command? wouldn't that be a comparable effort to documenting and updating the bash script?

File: deployments/relayer/build-path (hunk 1 above)
Comment (conorsch) [reply deployments/relayer/build-path:30]: We haven't been exercising these paths much! That's changing now, as IBC comes into focus.

File: docs/guide/src/pcli/transaction.md (hunk 2 above)
Comment (conorsch) [reply docs/guide/src/pcli/transaction.md:201]: Totally agree: enumerating all the matches to a query is way more useful cli functionality that spot-checking individual query targets. That's tracked in #2937.

**Pull request merged**